scripts/bundleContentScopeScripts.mjs

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs/promises'
```

Comment:
How come this is a `.mjs` (instead of `.js`) file?

Reply:
So we can use es6 imports with `node`.